apis/logging/v1/index_management_types.go

```diff
@@ -47,8 +47,25 @@ type IndexManagementPhasesSpec struct {
 // +k8s:openapi-gen=true
 type IndexManagementDeletePhaseSpec struct {
 	// The minimum age of an index before it should be deleted (e.g. 10d)
-	//
 	MinAge TimeUnit `json:"minAge"`
+	// The per namesapce specification to delete documents older than a given minimum age
+	Namespaces []IndexManagementDeleteNamespaceSpec `json:"namespaceSpec"`
+
+	// If MinAge for NamespaceSpec is empty, delete the records matching the namespaces which are older than this default age
+	DefaultAge TimeUnit `json:"defaultAge"`
```

Comment:
Why this `DefaultAge` here? Already inside [IndexManagementDeleteNamespaceSpec](https://github.com/openshift/elasticsearch-operator/pull/797/files#diff-228f46f0bf5d4ecce1a006430e1ed7172843e0b0ec24010b2bd919d4ab104de7R66-R68) it has been defined right? You can just check that empty and use the default age.